bin/webpack.js

```diff
@@ -167,6 +167,10 @@ if(argv._.includes('init')) {
 	const inputConfigPath = path.resolve(process.cwd(), filePaths[0]);
 
 	return require('../lib/migrate.js')(inputConfigPath, inputConfigPath);
+} else if(argv._.includes('generate-loader')) {
```

Comment:
If we were to support v4 this would be a thing that needs change. My self included, have used `.includes`, but I think that's v4<